packages/hooks/src/useFilteredCFDAccounts.ts

```diff
@@ -0,0 +1,48 @@
+import { useMemo } from 'react';
+import useAvailableMT5Accounts from './useAvailableMT5Accounts';
+import useExistingCFDAccounts from './useExistingCFDAccounts';
+
+/**
+ *
+ * @description This hook is to compare the available MT5 accounts with the existing CFD accounts and return the filtered list.
+ *
+ */
+const useFilteredCFDAccounts = () => {
+    const { data, isLoading } = useAvailableMT5Accounts();
+    const { data: existing_cfd_accounts, isLoading: existing_cfd_accounts_loading } = useExistingCFDAccounts();
+
+    const filtered_mt5_accounts = useMemo(() => {
+        if (!data) return undefined;
+
+        return Object.keys(data)
+            .map(key => {
+                const first_account = data[key][0];
```

Comment:
@thisyahlen-deriv if BE changes the order, this will break, better to find the `gaming` market_type and change it instead of modifying the first one.

Reply:
@mahdiyeh-deriv the order doesnt matter here actually. The order is actually jumbled up in the response.